cmd/demo/validator.go

```diff
@@ -93,3 +94,16 @@ func nonce() *big.Int {
 	}
 	return val
 }
+
+// etherToWei converts amount in "ether" (represented as float) to "wei" (represented as integer).
+// It can provide exact results for values in the range of 1e-18 to 1e9.
+func etherToWei(ether *big.Float) *big.Int {
+	weiFloat := new(big.Float).Mul(ether, new(big.Float).SetFloat64(params.Ether))
+	wei, _ := weiFloat.Int(nil) //acc was tested to be exact for the specified input range
```

Comment:
What is `acc`? Also, please put one space after `//` and end full sentences with a `.`.

Reply:
Addressed in this commit - 806b792.